packages/plugins/contracts/test/RevertDuringSimulation.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface ISphinxModule {
+    function activeMerkleRoot() external view returns (bytes32);
+}
+
+contract RevertDuringSimulation {
```

Comment:
I had to move this contract out of the `RevertDuringSimulation.s.sol` script file [due to this bug](https://linear.app/chugsplash/issue/CHU-407/deployment-fails-if-you-attempt-to-deploy-a-contract-which-is-in-the). 

This is actually a great example of a bug that would not currently be caught in our simulation logic since it only occurs b/c of the recompilation process. After this refactor, this bug started getting caught when we tried to execute this script with the `deploy` command in our tests so we have to move the contract to a different file to work around it.